mne/data/coil_def_Elekta.dat

```diff
@@ -39,6 +39,12 @@
 #			2	accurate
 #
 #
+1   2000    2   1  0.000e+00  0.000e+00	"Point magnetometer, x-normal"
```

Comment:
Numbers can be used multiple times, unfortunately. I wonder if this is a case where they did accidentally.